Scuti/Scripts/UI/Store/OfferSummaryPresenterBase.cs

```diff
@@ -100,17 +101,21 @@ public void LoadImage()
                             url = url.Insert(url.LastIndexOf("."), "_large");
                         if (DisplayAd)
                         {
-                            if (IsTall && !string.IsNullOrEmpty(TallURL))
+                            if(isSingle)
                             {
-                                url = TallURL;
-                            }
-                            else if (!IsTall && !string.IsNullOrEmpty(SmallURL))
-                            {
-                                url = SmallURL;
-                            }
-                            else
-                            {
-                                DisplayAd = false;
+                                if (IsTall && !string.IsNullOrEmpty(TallURL))
```

Comment:
Need to check if it is portrait mode here. Should only show TallURL in landscape. This was an existing bug.  @Jpablogs8

Reply:
Is better this?

if (IsTall && !string.IsNullOrEmpty(TallURL) )
{
        if(!ScutiUtils.IsPortrait())
               url = TallURL;

}

Reply:
Actually.... I just realized we don't need this.  We already check IsTall which is probably never true in portrait. 

lol

Sorry!  I'll strip this out and merge

Reply:
Yes, indeed istall always arrives False.